src/site/content/en/blog/responsiveness/index.md

```diff
@@ -0,0 +1,302 @@
+subhead: An update on our plans for measuring responsiveness on the web.
+description: An update on our plans for measuring responsiveness on the web.
+authors:
+  - hbsong
+date: 2021-11-03
+hero: image/eqprBhZUGfb8WYnumQ9ljAxRrA72/GNWyQZ5l1uy1b7s5ursN.jpeg
+alt: Water droplet rippling outward
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - performance
+  - web-vitals
+---
+
+Earlier this year, the Chrome Speed Metrics Team shared [some of the
+ideas](/better-responsiveness-metric/) we were considering for a
+new responsiveness metric. We want to design a metric that better captures the
+end-to-end latency of individual events and offers a more holistic picture of
+the overall responsiveness of a page throughout its lifetime.
+
+We've made a lot of progress on this metric in the last few months, and we
+wanted to share an update on how we plan to measure interaction latency as well
+as introduce a few specific aggregation options we're considering to quantify
+the overall responsiveness of a web page.
+
+We'd love to get [feedback](#feedback) from developers and site owners as to
+which of these options would be most representative of the overall input
+responsiveness of their pages.
+
+## Measure interaction latency
+
+As a review, the [First Input Delay (FID) metric captures the
+delay portion](/fid/#fid-in-detail) of input latency. That is, the time between
+when the user interacts with the page to the time when the event handlers are
+able to run.
+
+With this new metric we plan to expand that to capture the [full event
+duration](/better-responsiveness-metric/#capture-the-full-event-duration), from
+initial user input until the next frame is painted after all the event handlers
+have run.
+
+We also plan to measure
+[interactions](/better-responsiveness-metric/#group-events-into-interactions)
+rather than individual events. Interactions are groups of events that are
+dispatched as part of the same, logical user gesture (for example:
+`pointerdown`, `click`, `pointerup`).
+
+To measure the total interaction latency from a group of individual event
+durations, we are considering two potential approaches:
+
+- **Maximum event duration:** the interaction latency is equal to the largest
+  single event duration from any event in the interaction group.
+- **Total event duration:** the interaction latency is the sum of all event
+  durations, ignoring any overlap.
+
+As an example, the diagram below shows a _key press_ interaction that consists
+of a `keydown` and a `keyup` event. In this example there is a duration overlap
+between these two events. To measure the latency of the key press interaction,
+we could use `max(keydown duration, keyup duration)` or `sum(keydown duration,
+keyup duration) - duration overlap`:
+
+{% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/td8jMQBTjyR34ZO1QBvp.svg", alt="A
+diagram showing interaction latency based on event durations", width="800",
+height="424" %}
```

Comment:
I cleaned up this diagram. There were a lot of lines and stuff not lining up, some alignment issues, and I wanted to make it clearer what lines were referring to what.

Reply:
👍